test/xref_runner_SUITE.erl

```diff
@@ -0,0 +1,517 @@
+-module(xref_runner_SUITE).
```

Comment:
This test suite has nothing to do here. It tests the functions on `xref_runner`, which is *not* a part of your system.